packages/popup/src/LionPopup.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable class-methods-use-this, no-underscore-dangle */
```

Comment:
line should not be needed => underscore-dangle is globally disabled and class methods should use this... (default LitElement life cycle methods are already excluded) if you have another function where you don't need this it is better to exclude it specifically